### DISCUSSION TITLE
The provided link not respodding.

### DISCUSSION THREAD

**Discussion by Bone-Age-Maisha** [BCV-Uniandes/Bonet#7] [Q&A] [answered]
Bone Age Assessment Resources segment provided link are not responding. Is there any other way to get the dataset?

**Comment by mc-escobar11** [comment, marked as answer]
Thank you for your interest on the RHPE dataset, our webpage has been going through some modifications and is not available at the moment. However, you can access the datasets and annotations through the following links. If you need to evaluate the performance on the test set please send me an email (mc.escobar11@uniandes.edu.co) with a csv file following the same format of the examples ([RHPE](https://drive.google.com/file/d/1tAQW81d3PHmvrS1wPhGmmZZvyS5DrTuK/view?usp=sharing), [RSNA](https://drive.google.com/file/d/1vGF9BtXhzXlAf9Mq3jwEBmqi6JZx6uZf/view?usp=sharing)).
 
RHPE: [Readme](https://drive.google.com/file/d/1hxAV5trnxTbIxJwoH3UsSxPsSywpWRrF/view?usp=sharing)| [Train](https://drive.google.com/file/d/1Lq3XCOdOkJJKXPVPEYLvf1ScaFGh5uU6/view?usp=sharing)| [Validation](https://drive.google.com/file/d/1bjvC1YZ0h0TK1MXeamWXb8dF6wx-ZKQQ/view?usp=sharing)| [Test](https://drive.google.com/file/d/1sNFFDxbYjIeeBoSeYUokyq4dts_ZgnA_/view?usp=sharing)| [Annotations](https://drive.google.com/file/d/1NOtsunlmnYMufXEeeIHO_DPI77HJzx31/view?usp=sharing)
RSNA:[Train](https://drive.google.com/file/d/1SKmjs8Lvjv8DBs3LSy0XkbWDyJr_U3YF/view?usp=sharing)| [Validation](https://drive.google.com/file/d/17JM6Y5VeVgS35ywTKJ1U5CqHhDsF8irf/view?usp=sharing)| [Test](https://drive.google.com/file/d/1-8akTtn8rAf4cQklIJQ1NUbFnTlGocgc/view?usp=sharing)| [Annotations](https://drive.google.com/file/d/1mOveYqmsiiONFxAUhDCWaFvCnq6GSIRl/view?usp=sharing)
 
Please let me know if you have any other questions, I will try to reply as soon as possible.
